assets/quick-add.js

```diff
@@ -86,5 +87,24 @@ if (!customElements.get('quick-add-modal')) {
       galleryList.setAttribute('role', 'presentation');
       galleryList.querySelectorAll('[id^="Slide-"]').forEach(li => li.setAttribute('role', 'presentation'));
     }
+
+    updateImgSizes() {
+      const mediaGallery = this.modalContent.querySelector('[id^="MediaGallery-"]');
+      
+      if (mediaGallery.dataset.desktopLayout === 'columns') {
```

Comment:
Here I think instead of using data attributes we could maybe leverage the class names we already have and query mediaGallery with : `mediaGallery.classList.contains('product--columns')` and below `mediaGallery.classList.contains('product--medium')`